db/migrate/20170427152006_remove_invalid_hawkular_endpoints.rb

```diff
@@ -0,0 +1,32 @@
+class RemoveInvalidHawkularEndpoints < ActiveRecord::Migration[5.0]
+  class ExtManagementSystem < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+    has_many :endpoints, :as => :resource, :dependent => :destroy, :autosave => true
+  end
+
+  class Endpoint < ActiveRecord::Base; end
+
+  def up
+    Endpoint.where(
+      :resource_type => "ExtManagementSystem",
+      :role          => 'hawkular',
```

Comment:
They are using only the default[1]. 
Since it saves a join I prefer it.


[1] https://github.com/manageiq/manageiq-ui-classic/blob/deadce343f4105142a56e07cae3aab89a9d9151b/app/controllers/mixins/ems_common_angular.rb#L443-L446